samples/storage_csm_powerflex_v230.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: storage.dell.com/v1alpha1
+kind: ContainerStorageModule
+metadata:
+  name: test-vxflexos
+  namespace: test-vxflexos
+spec:
+  driver:
+    configVersion: v2.3.0
```

Comment:
do we need a 
`csiDriverType: "powerflex"`
in this section?

Reply:
added it